boards/nxp/imx943_evk/board.cmake

```diff
@@ -0,0 +1,4 @@
+#
+# Copyright 2025 NXP
+#
```

Comment:
file to go

Reply:
will remove, thanks.

Reply:
sorry, this file can't be removed as added jlink code.

Reply:
add it in the commit that uses it instead of adding a blank file in a commit?